crates/sui-indexer/migrations/pg/2024-09-12-213234_watermarks/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE watermarks
+(
+    -- The table name of group of tables governed by this watermark, i.e `epochs`, `checkpoints`,
+    -- `transactions`.
+	entity                      TEXT          NOT NULL,
+    -- Upper bound epoch range to enable per-entity epoch-level retention policy.
+    epoch_hi                    BIGINT        NOT NULL,
+    -- Lower bound epoch range to enable per-entity epoch-level retention policy.
+    epoch_lo                    BIGINT        NOT NULL,
+    -- The inclusive high watermark that the committer advances.
+	hi                          BIGINT        NOT NULL,
+    -- The inclusive low watermark that the pruner advances. Data before this watermark is
+    -- considered pruned.
```

Comment:
The "considered pruned" wording may be contributing to the confusion between this column and `pruned_lo`. Data below `lo` should be considered pruned by a reader, and data below `pruned_lo` should be considered pruned by the the pruner -- can we clarify that in the comments?